pkg/frontend/querymiddleware/querysharding_test.go

```diff
@@ -1643,6 +1643,102 @@ func TestQuerySharding_ShouldUseCardinalityEstimate(t *testing.T) {
 
 }
 
+func TestQuerySharding_Warnings(t *testing.T) {
+	storageSeries := make([]*promql.StorageSeries, 0)
```

Comment:
Is there a reason to use `make(..., 0)` here rather than declaring `storageSeries` following the same pattern as for `floats` below (ie. `var storageSeries []*promql.StorageSeries`)?

Reply:
Alternatively, we could use `numSeries` instead of `0` here.

Reply:
Made them consistent and declared capacities up front.